docs/eventing/sources/sinkbinding/_index.md

```diff
@@ -0,0 +1,38 @@
+---
+title: "SinkBinding"
+weight: 01
+type: "docs"
+aliases:
+  - /docs/eventing/samples/sinkbinding/index
+  - /docs/eventing/samples/sinkbinding/README
+  - /docs/eventing/sources/sinkbinding.md
+---
+
+![API version v1](https://img.shields.io/badge/API_Version-v1-red?style=flat-square)
+
+The `SinkBinding` custom object supports decoupling event production from
+delivery addressing.
+
+You can use a SinkBinding to direct a subject to an event sink. The _subject_
+is a Kubernetes resource that embeds a `PodSpec` template and produces events.
+The _event sink_ is an addressable Kubernetes object that can receive events.
+
+To create new subjects you can use any of the compute objects that Kubernetes
+makes available such as:
+
+- `Deployment`
+- `Job`
+- `DaemonSet`
+- `StatefulSet`
+
+You can also create new subjects that use Knative abstractions, such as
+`Service` or `Configuration` objects.
```

Comment:
These are still custom resources, just for Knative rather than Kubernetes, but they're all addressable Kubernetes objects, so I'd maybe reword this a bit to make it clearer. I'm actually not really sure it's important to call all of this out because there are a ton of things that can be used for sinks.

Out of curiosity, why use the word _subject_ above instead of just _sink_ consistently?

Reply:
About the list:
I'll remove this list as it is also mentioned on the "Create a SinkBinding" page.

About the word subject:
I worked with an engineer on this who said that "subject" was a better word than "source". I thought it was reasonable since the command to create the SinkBinding uses the flags `--subject` and a `--sink`, as in:

```
kn source binding create <name> \
  --namespace <namespace> \
  --subject "<subject>" \
  --sink <event-sink>
```

Do you disagree?